rmgpy/rmg/model.py

```diff
@@ -687,6 +687,13 @@ def enlarge(self, newObject=None, reactEdge=False,
             logging.info('Generating thermo for new species...')
             self.applyThermoToSpecies(procnum)
 
+        # Do thermodynamic filtering
+        try:
+            if not numpy.isinf(self.toleranceThermoKeepSpeciesInEdge) and spcs != []:
+                self.thermoFilterSpecies(spcs)
+        except:
```

Comment:
What is the exception that this is expecting?

Reply:
correctly defining self.newSpeciesList allows the removal of try and except, thanks!